templates/v2/common/_card_with_form.html.twig

```diff
@@ -1,14 +1,19 @@
 {% extends 'v2/common/_card.html.twig' %}
 
+{% set passwordFields = ['plainPassword', 'password'] %}
+
 {% block customContent %}
     <div class="text-start">
         {{ form_start(form, {attr: {class: 'display-required-labels'}}) }}
         <div class="row text-primary">
             {% for field in form %}
-{#          The 'plainPassword' field name is only used when creating a professionnal #}
-                {% if field.vars.name == 'plainPassword' %}
+                {% if field.vars.name in passwordFields %}
                     {{ form_errors(field) }}
-                    {{ include('v2/user/strong_password/_strong_password_row.html.twig', {'field' : field, 'isBeneficiaire': false, 'hideAndShow': false }) }}
+                    {% if isBeneficiary is defined and isBeneficiary %}
```

Comment:
There seem to be a lot of user specific code in a template called `common/card_with_form` that looks 100% generic